partials/forms/input-text-group.html

```diff
@@ -1,4 +1,4 @@
-<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{^date}}{{#error}} error{{/error}}{{/date}}">
+<div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{^date}}{{#error}} error{{/error}}{{/date}} " {{#attributes}} {{attribute}}="{{value}}" {{/attributes}}>
```

Comment:
Yeah think groupAttributes is clearer. If possible it would be good to add the groupAttributes to the text input groups as well as the radios/selects. For example, it makes the GA tracking easier as the attributes are always on the wrapping div.